### ISSUE TITLE
[Machine Learning] Building a Multiclass image classification model using MobileNet-V2 and TensorFlow

### ISSUE THREAD

**Issue by kelivinnganga** [section-engineering-education/engineering-education#6758]
#### Proposed title of article
Building a Multiclass image classification model with MobileNetV2 and TensorFlow

#### Proposed article introduction
Image classification categorizes input images into pre-defined labels or categories. Most image classifications usually have only two classes. In a multi-class, we have three or more classes such as `orange`, `mango`, `apple`, and `pineapple`. Multi-class image classification categorizes an input image into one of the three or more classes. A model is trained on a set of image datasets so that it can be able to classify images into pre-defined labels. Multiclass image classification is a common task in computer vision.

MobileNetV2 is a type of pre-trained convolutional neural network based on a streamlined architecture that uses convolutions to build light weight deep neural networks that can have low latency for mobile and embedded devices. MobileNets support any input size greater than 32 x 32, with larger image sizes offering better performance.

Pre-trained models are neural networks which are trained on very large datasets, developed, and are used to solve similar types of problems. We will use the pre-trained Mobile Net model to build a model that classifies various leaf images.

#### Key takeaways 
1. Application of transfer learning.
2. Downloading and customize the pre-trained MobileNet-v2.
3. Images standardization and scaling.
4. Fine-tuning the model.
5. Classification of various leaf images.
#### Article quality
This article is unique because it will explain how to download the pre-trained MobileNet-v2 from TensorFlow hub and customizing it to perform multi-class image classification. We also discuss how to fine-tune the model by adding custom Keras layer to improve the models performance. We will also discuss the important aspect of image preprocessing so that we provide the required image format for model training.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by LinusMuema** [comment]
good topic @kelivinnganga - approved. Thanks for submitting your article and we look forward to reviewing and hopefully publishing it. I’m sure our readers will enjoy and find your article very beneficial.
